config/config.go

```diff
@@ -478,6 +482,10 @@ func NewConfig() *Config {
 	flag.Var(cfg.CredentialPaths, "credentials-paths", "directories or files to watch for credentials to use by bearerinjector filter")
 	flag.DurationVar(&cfg.CredentialsUpdateInterval, "credentials-update-interval", 10*time.Minute, "sets the interval to update secrets")
 
+	// Open Policy Agent
```

Comment:
I think OPA should be part of Auth, so drop comment and move up the lines.

Reply:
You can also argue that it is not: one of the two filters that is introduced is for serving requests, not authz. 

Your call, I am happy with either and agree that many people would file it under auth

Reply:
Moved it to auth